app/views/books/_filter.html.haml

```diff
@@ -2,11 +2,11 @@
 %ul.list-inline.pt-10.mb-25.mr-240
   %li.mr-35
     %a.filter-link{href: books_path}
-      All
-      %span.badge.general-badge= Book.count
+      = t('shop.all')
+      %span.badge.general-badge= @book_count
   %li.mr-35
     %a.filter-link{href: "#"}
-      - Category.includes(:books).find_each do |category|
+      - @categories.each do |category|
         %li.mr-35
           .filter-link
             = link_to category.name, books_path(category: category.id), class: 'collapse-link'
```

Comment:
Seems like this link follow me to books#show with the id of a selected category. Is it expected behavior?